### DISCUSSION TITLE
Imports

### DISCUSSION THREAD

**Discussion by MixusMinimax** [sbrunner/jsonschema-gentypes#922] [Q&A] [answered]
Does this converter support imports to other files? As in, I have a base-schema that I refer to from other schemas, and I want to create .py files for all of them, and have them import types from the base .py file.

**Comment by sbrunner** [comment, marked as answer]
No, currently nothing like that is supported...

**Comment by MixusMinimax** [comment]
Alright no worries, typed dicts are kinda like typescript interfaces anyway so I can pass variables of those types to functions that expect another type without any warnings.
Does the job either way, it's definitely much better than completely untyped dicts!
